localgov_microsites_group.install

```diff
@@ -17,6 +17,12 @@ function localgov_microsites_group_install($is_syncing) {
     return;
   }
 
+  // Group sites should be installed aleady. It provides configuration, so we
+  // can't, but must update its default config.
+  $group_sites = \Drupal::configFactory()->getEditable('group_sites.settings');
+  $group_sites->set('context_provider', '@group_context_domain.group_from_domain_context:group');
```

Comment:
Not related to the install line. That was actually missing from the upgrade, so nice to have it highlighted. The dupe plugin is because one is deprecated. Changing the title of that so it now shows

![image](https://github.com/localgovdrupal/localgov_microsites_group/assets/237186/e4f3126d-ef77-4a3c-996f-697f2801ed14)